ads/inabox/inabox-host.md

```diff
@@ -14,4 +14,87 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-This is still in development. Check [here](https://github.com/ampproject/amphtml/issues/5700) for details. 
\ No newline at end of file
+# SDK integration for AMPHTML ad
+One goal of AMPHTML ad is for advertisers to create ad once and use 
+every where. This guide provides details about how an ad network can 
+join us to serve and render AMPHTML ad in various environment, for 
+example regular web & native mobile apps.
+
```

Comment:
Mention server-side rendering somewhere?

Reply:
Hesitated to do so... given it's not really standardized at this moment. It makes quite lot assumption with AMP runtime. There is an on-going effort to open-source some SSR for regular AMP doc. But I don't think it's that ready yet, as nothing about amp4ads is covered.

Reply:
I think it's worth mentioning that this is in progress, and that if you're looking at serving inabox you should get in touch with the project.  Without SSR performance is pretty bad.

Reply:
added a section for `Server side rendering`